app-frontend/src/app/components/newProjectModal/newProjectModal.component.js

```diff
@@ -0,0 +1,15 @@
+// Component code
+import newProjectModalTpl from './newProjectModal.html';
+
+const NewProjectModalComponent = {
```

Comment:
@alkamin I put this in as an empty stub but if it'd reduce merge conflicts for you I can take it out before merging.